cl/singlenode/singlenode.go

```diff
@@ -18,13 +18,26 @@ import (
 	"github.com/primev/mev-commit/cl/singlenode/payloadstore"
 	localstate "github.com/primev/mev-commit/cl/singlenode/state"
 	"github.com/primev/mev-commit/cl/types"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 const (
 	// Stop Function
 	shutdownTimeout = 5 * time.Second
 )
 
+var (
+	snDBDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
The promauto pkg registers the collectors also?

Reply:
Yes promauto automatically registers the metrics it creates with the default registry, and `promhttp.Handler()` uses the default registry